packages/vite-node/src/client.ts

```diff
@@ -187,7 +187,10 @@ export class ViteNodeRunner {
       __vite_ssr_dynamic_import__: request,
       __vite_ssr_exports__: exports,
       __vite_ssr_exportAll__: (obj: any) => exportAll(exports, obj),
-      __vite_ssr_import_meta__: { url },
+      __vite_ssr_import_meta__: {
+        url,
+        hot: this.options.createHotContext?.(this, `/@fs/${fsPath}`),
```

Comment:
Can this be a getter? I am a bit worried about the performance impact on it.

Reply:
createHotContext when transformd code had `__vite_ssr_import_meta__.hot`?